crates/rpc/src/handlers/block.rs

```diff
@@ -272,3 +279,46 @@ pub async fn get_block_from_id(
 
     Ok(HttpResponse::Ok().json(BeaconVersionedResponse::new(beacon_block)))
 }
+
+/// Called by `/beacon/heads` to get fork choice leaves.
+#[get("/beacon/heads")]
+pub async fn get_beacon_heads(db: Data<ReamDB>) -> Result<impl Responder, ApiError> {
+    let base = db
+        .justified_checkpoint_provider()
+        .get()
+        .map_err(|_| ApiError::InternalError)?;
+    let root = base.root;
```

Comment:
```suggestion
    let justified_checkpoint = db
        .justified_checkpoint_provider()
        .get()
        .map_err(|_| ApiError::InternalError)?;
```

please print a log, to expose the error being passed